### PR TITLE
gh-89653: PEP 670: amend docs

### PR DESCRIPTION
**Pull request by erlend-aasland**

### CODE REVIEW

File: Doc/c-api/unicode.rst [hunk 1]
```diff
@@ -172,9 +172,9 @@ access internal read-only data of Unicode objects:
                                      Py_UCS4 value)
 
    Write into a canonical representation *data* (as obtained with
```
Comment (vstinner) [inline Doc/c-api/unicode.rst:174]: PyUnicode_WRITE() kind parameter type is unsigned int, as PyUnicode_KIND() result in the doc. Can you please update it? Same remark for PyUnicode_READ().

**Review by vstinner — COMMENTED** [review]
LGTM, but please fix kind parameter type in the two mentioned functions doc.

Oh, I forgot that the doc mentioned that some functions are implemented as macros. Thanks for updating the doc!

File: Doc/c-api/unicode.rst (hunk 1 above)
Comment (erlend-aasland) [reply Doc/c-api/unicode.rst:174]: Well spotted! Thanks, I've updated the PR.

**Review by vstinner — APPROVED** [review]
LGTM.

**Comment by vstinner** [comment]
Merged, thanks.

**Pull request merged**